raiden/api/python.py

```diff
@@ -370,6 +375,14 @@ def channel_open(
             )
 
         token_network = self.raiden.chain.token_network(registry.get_token_network(token_address))
+        given_block_identifier = views.state_from_raiden(self.raiden).block_hash
+
+        if self.channel_already_exists(
```

Comment:
Can you please change this to:
```
channel_exists = self.is_channel_already_exists()
if channel_exists():
    ....
```

We usually prefer to call the function outside an if-statement and use the result var with `if`